.github/workflows/lighthouse.yml

```diff
@@ -0,0 +1,50 @@
+name: Lighthouse CI
+
+on:
```

Comment:
I think that this workflow might also benefit from trigger [workflow_dispatch](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch). In case this action fails for any reason, there will be a way for a maintainer to restart it by hand.

Reply:
In case of fail re-run option should be available on the run page, e.g. https://github.com/allegro/turnilo/runs/5303537047?check_suite_focus=true

Reply:
What about a case, when every action in a workflow reports OK, and yet some side effect failed? There is no way to retry it

Reply:
I've added `workflow_dispatch`